operations/bookings.md

```diff
@@ -253,9 +260,27 @@ A booking represents a reservation made by a booking for goods or services, such
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `type` | string | required | The [type](https://jsonapi.org/format/#document-resource-object-identification) member is used to describe resource objects that share common attributes and relationships. |
-| `attributes` | [booking_attributes](bookings.md#booking_attributes) | required | An [attributes object](https://jsonapi.org/format/#document-resource-object-attributes) representing some of the resource's data. |
+| `attributes` | [booking_create_attributes](bookings.md#booking_create_attributes) | required | An [attributes object](https://jsonapi.org/format/#document-resource-object-attributes) representing some of the resource's data. |
 | `relationships` | object | required | An [relationships object](https://jsonapi.org/format/#document-resource-object-relationships) representing associations with other resources. |
 
+#### booking_create_attributes
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `createdAt` | string | required, max length 25 characters | Created at timestamp in RFC 3339 format. |
+| `updatedAt` | string | required, max length 25 characters | Updated at timestamp in RFC 3339 format. |
+| `required` | undefined | required |  |
+| `status` | string,null | optional | The initial status of the booking. Possible values are "confirmed", "seated", "completed", "cancelled", and "noShow". |
+| `partySize` | integer | required | Represents the number of people included in the booking. |
+| `bookingDatetime` | string | required, max length 25 characters | The booking's date. |
+| `duration` | integer,null | optional | Represents the length of the booking in minutes. |
+| `notes` | string,null | optional, max length 10000 characters | Additional notes for the booking. |
+| `roomNumber` | string,null | optional, max length 100 characters | The room number of the booking's customer. |
+| `promotions` | string,null | optional, max length 100 characters | The promotions of the booking. |
+| `bookingReference` | string,null | optional, max length 255 characters | A reference code or identifier associated with the booking. |
+| `isWalkIn` | boolean | required | Indicates if the booking is a walk-in. |
```

Comment:
@MirkoC I just noticed this as well. Should `isWalkIn` be _required_ or should it simply default to `false`?
* If _required_ then the sample JSON is invalid because it is not present
* If NOT _required_ then we should update the property entry to this ->
```suggestion
| `isWalkIn` | boolean | optional | Indicates if the booking is a walk-in. Defaults to `false`. |
```

Reply:
@MikeAdamsMews I'm not sure, I haven't worked on this part myself as much. @joseluis-rodriguez do you know?

Reply:
@MikeAdamsMews I'm not sure that we need to do anything with this. As these `booking_create_attributes` are response attributes, not request body attributes. And in response `isWalkIn` will always be there (and will not default to `false`).

Reply:
@MirkoC the `booking_create_attributes` are **request** attributes _not_ **response** attributes.